Neos.Flow/Classes/ObjectManagement/Proxy/ObjectSerializationTrait.php

```diff
@@ -116,7 +123,7 @@ private function Flow_searchForEntitiesAndStoreIdentifierArray(string $path, mix
                 $this->Flow_searchForEntitiesAndStoreIdentifierArray($path . '.' . $key, $value, $originalPropertyName);
             }
         } elseif ($propertyValue instanceof PersistenceMagicInterface && !Bootstrap::$staticObjectManager->get(PersistenceManagerInterface::class)->isNewObject($propertyValue) || $propertyValue instanceof DoctrineProxy) {
-            if (!property_exists($this, 'Flow_Persistence_RelatedEntities') || !is_array($this->Flow_Persistence_RelatedEntities)) {
+            if (!property_exists($this, 'Flow_Persistence_RelatedEntities') || !isset($this->Flow_Persistence_RelatedEntities) || !is_array($this->Flow_Persistence_RelatedEntities)) {
```

Comment:
It seems we don’t need `property_exists` anymore ;)

and we should def look up if property_exists is used elsewhere ein the codebase (as it might not work anymore)